config/sewing-kit/index.ts

```diff
@@ -11,7 +11,7 @@ import type {} from '@sewing-kit/plugin-jest';
 
 const PLUGIN = 'ArgoCheckout.DefaultPackage';
 
-export function argoCheckoutPackage({react: useReact = false} = {}) {
+export function argoPackage({react: useReact = false} = {}) {
```

Comment:
Why the name change?

Reply:
Because we're using the same util for Admin now